src/stan/math/prim/scal/err/check_bounded.hpp

```diff
@@ -4,6 +4,8 @@
 #include <stan/math/prim/scal/err/domain_error.hpp>
 #include <stan/math/prim/scal/err/domain_error_vec.hpp>
 #include <stan/math/prim/scal/meta/max_size.hpp>
+#include <stan/math/prim/scal/meta/is_vector_like.hpp>
+#include <stan/math/prim/arr/meta/is_vector_like.hpp>
```

Comment:
remove this line. headers in `prim/scal` should not include things in `prim/arr`.